test/integration/testExecute.js

```diff
@@ -431,5 +431,88 @@ describe('Execute test - variant', function () {
 
     it(testCase.name, createItCallback(testCase, rowAsserts));
   });
-});
 
+  describe.only( 'connection.execute() Resubmitting requests using requestId and different connections', function () {
+    const createTable = 'create or replace table test_request_id(colA string)';
+    let  firstConnection;
+    let  secondConnection;
+    before(async () => {
+      firstConnection = testUtil.createConnection();
+      secondConnection = testUtil.createConnection();
+      await testUtil.connectAsync(firstConnection);
+      await testUtil.connectAsync(secondConnection);
+      await testUtil.executeCmdAsync(firstConnection, 'truncate table if exists test_request_id');
```

Comment:
do we need truncate just before create?

Reply:
It isn't obligatory when we execute "create or replace" in the next step. Thank you